docs/changelog/CHANGELOG.md

```diff
@@ -43,6 +43,7 @@ See also [LTS releases](https://docs.victoriametrics.com/lts-releases/).
 * BUGFIX: [vmui](https://docs.victoriametrics.com/#vmui): prevent accordion from collapsing when selecting text in headers. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/7742).
 * BUGFIX: [vmui](https://docs.victoriametrics.com/#vmui): fix query interval display in the instant query info. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/7401).
 * BUGFIX: [vmalert](https://docs.victoriametrics.com/vmalert): fix possible template collision between rule annotations.
+* BUGFIX: [vmsingle](https://docs.victoriametrics.com/single-server-victoriametrics/), `vmselect` in [VictoriaMetrics cluster](https://docs.victoriametrics.com/cluster-victoriametrics/): make searching a single sample with millisecond precision consistent. Previously searching a sample with 1ms step caused extending the search window to 5ms due to added jitter. The fix does not add jitter if the search result contains only one sample. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/5796) for details.
```

Comment:
Please replace the `jitter` word with `adjustment for small intervals`, with the link to the original issue, where this adjustment was added. The `jitter` word is misleading - it means adding or subtracting some random value to the original interval in order to avoid resource usage spikes when manu concurrent requests with hit the service.

Also it looks like the new code just updates the default interval from from the `step` to `step+5m`, when the interval between raw samples cannot be detected because of lack of raw samples. This means that the issue remains, but now it is visible when the distance between raw samples exceeds 5 minutes. I'm not sure this is a good fix.

Reply:
Changed the description to reflect the updated solution.